ui/src/components/ChangeLog/ChangeLog.tsx

```diff
@@ -0,0 +1,18 @@
+
+import './ChangeLog.sass';
+import { CollapseButton } from '../../useCommonStyles';
+
+export interface Props {
+    onCollapseAll: () => void
+}
+
+export const ChangeLog: React.FC<Props> = ({ onCollapseAll }) => {
```

Comment:
what is the meaning of the ChangeLog name of the component?
it seems like you have only title and collapse button here

Reply:
Yes basically its just a container.
What would you suggest?